src/main/java/com/autotune/analyzer/experiment/ExperimentInterfaceImpl.java

```diff
@@ -50,9 +57,27 @@ public boolean addExperimentToLocalStorage(Map<String, KruizeObject> mainKruizeE
 
     @Override
     public boolean addExperimentToDB(KruizeObject kruizeObject) {
```

Comment:
Looking at these functions, I think these need to be in a `ExperimentDBInterface` inside the `database` package. We can then have a `postgresDBImpl` that can host all of these functions.